doc: update some dynamic parameters from acrn-dm usage (#7041)
projectacrn/acrn-hypervisor: doc/user-guides/acrn-dm-parameters.rst

----

> ``--cpu_affinity <list of lapic_id>``

Reviewer: ```suggestion
``--cpu_affinity <list of lapic_ids>``
```

Author: Updated.

>    list of vCPUs assigned to this VM, the vCPUs are identified by their lapic IDs.

Reviewer: ```suggestion
   comma-separated list of vCPUs assigned to this VM. A vCPU is identified by its lapic ID.
```

Author: Updated.

>    Lapic IDs can use ``apicid`` in Service VM's ``/proc/cpuinfo``.

Reviewer: I think this needs more information.  What's the difference between apicid and lapic_id?

Reviewer 2: lapic_id is short for 'local apic id', is consistent with SDM , and is more precise. 
'apicid' is the description in ``/proc/cpuinfo`` in linux OS.

Do you have some suggestion for more information?

Reviewer 2: Updated， please check if it's easier to understand.

   Example::

      --cpu_affinity 1,3

>    to assign vCPUs with apicid 1 and 3 to this VM.

Reviewer: ```suggestion
   to assign vCPUs with lapic_id 1 and 3 to this VM.
```

Author: Updated.

----

``--cpu_affinity <list of lapic_ids>``
   comma-separated list of vCPUs assigned to this VM. Each CPU has a Local Programmable

>    Interrupt Controller(LAPIC). The unique ID of LAPIC(lapic_id) is used to identify vCPU.

Reviewer: ```suggestion
   Interrupt Controller (LAPIC). The unique ID of the LAPIC (lapic_id) is used to identify vCPU.
```

Reviewer 2: updated

>    The ``lapic_id`` for vCPU can be read from ``/proc/cpuinfo`` in service VM.

Reviewer: ```suggestion
   The ``lapic_id`` for a vCPU can be found in the Service VM file ``/proc/cpuinfo``, 
```

Reviewer 2: updated

>    In that file ``lapic_id`` is marked as ``apicid``.

Reviewer: ```suggestion
   identified as ``apicid``.
```

Reviewer 2: updated
